portal-web/test/functional/com/liferay/portalweb/macros/layout/contentpage/ContentPages.macro

```diff
@@ -4,11 +4,15 @@ definition {
 		if (isSet(collectionName)) {
 			Navigator.gotoNavNested(navNested = "${collectionName}");
 
+			WaitForLiferayEvent.initializeLiferayEventLog();
```

Comment:
I see we're still doing this... can we prioritze improving the way we collect and analyze events to remove all additional initializations, please?

Reply:
Hi @jbalsas,
I added this to fix IE11 failures when adding pages. This error blocks another ticket, so used it as a temporary solution to make the tests keep running after adding page. I will investigate more about how to fix this in a better way and then remove the initialization, but I am afraid that may take some time.

Regards!

Reply:
That's okay @zhangyan0701... just feels like one of those things that never get done so we're always walking on eggshells over a flaky test infrastructure and setup.

Let's try to put some time into this when we can!